packages/api/src/ITooltip.ts

```diff
@@ -123,10 +123,25 @@ export abstract class ITooltip extends Widget {
         }
     }
 
+    tooltipKeyValueFormat(obj) {
+        const _key_color = this.tooltipLabelColor_exists() ? this.tooltipLabelColor() : this.tooltipSeriesColor();
+        const _value_color = this.tooltipValueColor();
+        switch (this.tooltipStyle()) {
+            case "none":
+                break;
+            default:
+                return Object.keys(obj).map(key => _keyValueSpans(key, obj[key])).join("<br/>");
+        }
+        function _keyValueSpans(_key, _val) {
+            return `<span style='color:${_key_color}'>${_key}</span>:  <span style='color:${_value_color}'>${_val}</span>`;
+        }
+    }
```

Comment:
There is a similar implementation to this here:  https://github.com/hpcc-systems/Visualization/blob/master/demos/imdb/src/IMDBVertex.ts#L70-L86 
With better alignment - I am not sure you can support the "title" feature in the general case though?

Reply:
@GordonSmith I've updated the PR with your alternative function. It does look nicer.